openpower/configs/linux/skiroot_defconfig

```diff
@@ -216,7 +216,6 @@ CONFIG_I2C=y
 CONFIG_I2C_CHARDEV=y
 # CONFIG_I2C_HELPER_AUTO is not set
 CONFIG_I2C_ALGOBIT=y
-CONFIG_I2C_OPAL=y
```

Comment:
? is this because a previous defconfig change wasn't regenerated?

Reply:
It looks like so, but I'll double-check

Reply:
Yes, someone forgot to re-generate the skiroot defconfig on a kernel update as it became a default